src/java.desktop/windows/classes/sun/awt/windows/WPrinterJob.java

```diff
@@ -2297,7 +2297,7 @@ private void setPrinterNameAttrib(String printerName) {
     }
 
 @SuppressWarnings("serial") // JDK-implementation class
-class PrintToFileErrorDialog extends Dialog implements ActionListener{
+ static class PrintToFileErrorDialog extends Dialog implements ActionListener{
```

Comment:
There's an extra space before `static` modifier which breaks the common indentation.
I'd also add a space before the opening brace, it's there in the following code of the class.

Reply:
fixed